app/views/drawings/_form.html.haml

```diff
@@ -9,31 +9,33 @@
           = image_tag preview_image(@drawing), id: 'image-preview', class: 'img-responsive'
           %br
           %b Allowed image types: jpg/jpeg, png, tiff, gif, bmp
-          = f.input :image, label: false, required: true, input_html: { onChange: 'loadFile(event)' }
+          = f.input :image, label: false, required: false, input_html: { onChange: 'loadFile(event)' }
 
       .col-md-7.col-sm-12.col-xs-12
         = f.error_notification
         %h4 Details of Drawing (required when marked complete)
         .form-group
           = f.input :country, priority: TOP_COUNTRIES, selected: (@drawing.new_record? ? current_user.country : @drawing.country), label: "Country of Origin", include_blank: "Country picture was drawn in", class: 'form-control'
         .form-group
-          = f.input :subject_matter, collection: ["Home / Country of origin", "Transit", "Camp life", "Future hopes / destination"], label: false, required: true, prompt: "Subject matter of drawing"
+          = f.input :subject_matter, collection: ["Home / Country of origin", "Transit", "Camp life", "Future hopes / destination"], label: 'Subject matter of drawing', required: false, prompt: "e.g. Home / Country of Origin OR Camp Life"
```

Comment:
Not major, but the prompt here can make it a little uncertain whether I've selected something or not. In this instance I would use "Select an option", or "--select an option--" for the unselected state rather than an e.g. listing one or more of the selections.

![image](https://cloud.githubusercontent.com/assets/4599695/17903494/9666869a-6963-11e6-940b-4ef8e7debb50.png)